src/scss/pages/_home.scss

```diff
@@ -0,0 +1,117 @@
+@import "./utils/_variables";
+
+.appbar {  
+  background-color: $primary-color;
+}
+.nav-items {
+  padding: 0;
+  float: right;
+  padding: 5px 0;
+  margin-top: 0;
+
+  .nav-item {
+    display: inline-block;
+
+    .nav-link {
+      padding: 20px;
+      color: #ffffff;
+      text-decoration: none;
+      display: block;
+      border-bottom: 2px solid transparent;
+    }
+
+    .nav-link:not(.dropdown-button):hover {
+      border-bottom: 2px solid #ffffff;
+    }
+  }
+}
+.dropdown-toggle:hover {
+  .dropdown-content {
+    display: inline-block;
+  }
+}
+.dropdown-toggle {
+  position: relative;
+
+  .dropdown-content {
+    box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+    position: absolute;
+    display: none;
+    text-decoration: none;
+    list-style: none;
+    min-width: 160px;
+    padding-left: 0;
+    top: 100%;
+    left: 0;
+  }
+
+  .triangle-down {
+    width: 0;
+    height: 0;
+    border-left: 5px solid transparent;
+    border-right: 5px solid transparent;
+    border-top: 5px solid #ffffff;
+    display: inline-block;
+    position: relative;
+    top: -2px;
+  }
+
+  .dropdown-item:hover {
+    background-color: $primary-color;
+    color: #ffffff;
+  }
+
+  .dropdown-item {
+    background-color: #ffffff;
+    display: block;
+    padding: 12px 16px;
+  }
+
+  .dropdown-button:hover {
+    border-bottom: 2px solid transparent;
+  }
+}
+.contain{
```

Comment:
- tên class là danh từ nhé --> container
- làm thế này ko tốt lắm. container này dùng ở cả header lẫn phần nội dung ở dưới của e
=> có thể move vào trong _grid
=> nên set margin left-right 2 bên thay vì padding tính toán như kia